src/inc/msquic.h

```diff
@@ -817,6 +835,7 @@ typedef struct QUIC_SCHANNEL_CONTEXT_ATTRIBUTE_EX_W {
 #define QUIC_PARAM_STREAM_0RTT_LENGTH                   0x08000001  // uint64_t
 #define QUIC_PARAM_STREAM_IDEAL_SEND_BUFFER_SIZE        0x08000002  // uint64_t - bytes
 #define QUIC_PARAM_STREAM_PRIORITY                      0x08000003  // uint16_t - 0 (low) to 0xFFFF (high) - 0x7FFF (default)
+#define QUIC_PARAM_STREAM_BLOCKED_TIMINGS               0X08000004  // QUIC_FLOW_BLOCKED_TIMING[QUIC_FLOW_BLOCK_REASON_COUNT]
```

Comment:
Let's go ahead and make a `QUIC_STREAM_STATISTICS` struct and put the flow blocked timings in there, and have this return the stats.

Reply:
QUIC_STREAM_STATISTICS will only have blocked timings? Should I add other stats now in this PR?

Reply:
Done